frontend/src/components/common/TooltipWrapper.tsx

```diff
@@ -1,9 +1,26 @@
 import * as React from 'react';
 import { Tooltip, OverlayTrigger, OverlayTriggerProps } from 'react-bootstrap';
 
+/**
+ * TooltipWrapper properties.
+ * @interface ITooltipWrapperProps
+ * @extends {Partial<OverlayTriggerProps>}
+ */
 interface ITooltipWrapperProps extends Partial<OverlayTriggerProps> {
-  toolTip?: string;
-  toolTipId: string;
+  /**
+   * The tooltip text to display when hovering over the component.
```

Comment:
Adds intellisense to VS Code!  Please start documenting more of your components!